src/resources.rs

```diff
@@ -66,7 +66,7 @@ enum Repr {
   TcpStream(tokio::net::TcpStream),
   HttpBody(HttpBody),
   Repl(Repl),
-  Child(tokio_process::Child),
+  Child(Box<tokio_process::Child>),
```

Comment:
Can you add the a comment here about why Box. Otherwise looks good.

Reply:
done